### ISSUE TITLE
Org unit - Assign employee checkboxes do not have label

### ISSUE THREAD

**Issue by vanpham-sw** [shopware/platform#7817]
### PHP Version

8.2.25

### Shopware Version

6.7

### Affected area / extension

Extension:Commercial

### Actual behaviour

Without label, when using voice over to read these checkboxes, they are read as checkbox. 

![Image](https://github.com/user-attachments/assets/7519fbba-f4d1-4371-961e-51f2c0866ace)

### Expected behaviour

The label should be added so that voice over reads the checkboxes with employee name

### How to reproduce

1. Create an organization unit
2. Use voice over to assign employee
